src/Symfony/Component/ErrorHandler/Attribute/LogLevel.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\Attribute;
```

Comment:
I wasn't sure if this attribute would belong to this component but it made a bit more sense to me to place it here as it can be considered a general one.

Reply:
HttpKernel would also be a natural fit since that's where the listener is

Reply:
I was wondering if this attribute could be later used in places outside the HttpKernel, in which case it'd make more sense to me to be in this component? But if not, it can be moved to the HttpKernel.

Reply:
I'd vote for moving at this stage.